src/Features/LanguageServer/Protocol/Features/Options/DiagnosticModeExtensions.cs

```diff
@@ -1,26 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis.Options;
-
-namespace Microsoft.CodeAnalysis.Diagnostics
-{
-    internal static class DiagnosticModeExtensions
-    {
-        public static DiagnosticMode GetDiagnosticMode(this IGlobalOptionService globalOptions, Option2<DiagnosticMode>? option = null)
```

Comment:
both of these methods were basically inlined to always return DIagnosticMode.LspPull.